app/assets/javascripts/application/discussions.js

```diff
@@ -0,0 +1,43 @@
+var mumuki = mumuki || {};
+
+mumuki.load(function () {
+  var $newDiscussionModal = $('.new-discussion-modal');
+  var $newDiscussion = $('#discussion-create');
+
+  $newDiscussion.click(function () {
+    $newDiscussionModal.modal({
+      backdrop: 'static',
+      keyboard: false
+    });
+  });
+
+  var $subscriptionSpans = $('.discussion-subscription > span');
+  var $upvoteSpans = $('.discussion-upvote > span');
+
+  var Forum = {
+    toggleButton: function (spans) {
+      spans.toggleClass('hidden');
```

Comment:
Side note: If you already feel more comfortable with ES6, we are already supporting it, so go ahead and use it in the future for const, class declarations and so on.

Reply:
(No need to change it now)